### DISCUSSION TITLE
Make chromie store Mounts/Pets account wide

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3234] [Suggestions]
As discussed in the old suggestion channel, reward catalogue mounts and pets should be account wide.

The mounts are expensive, with the cheapest being £10 and going all the way up to £60 for celestial steed (£15 beck on retail btw). The blizzard battle bear is even account bound and still priced the same as the others

 I know chromie points are meant to be a donation / contribution award, but there should still be decent value when redeeming.
Better value means more donations which is ultimately better for the project. I definitely wouldve bought more mounts if they were cross character, but as it stands I regret buying one.

The way it worked back in the day was that they were mailed to every character, including future characters you made. I don't know how hard it would be to do future characters, but it should at least be doable to mail to all current characters

**Comment by 55Honey** [comment]
Hello Rubadubscrub,
Thanks for your suggestion.

ChromieCraft is a server which relies entirely on cosmetic items to generate income, which is in turn 100% invested into the server and the development.

While we would love to make more items accessible to more players, we also need to make sure that our income does not drop to a point where we can not pay for fixes anymore.

Everyone can receive ChromiePoints as a reward for their contributions to be able to get more items from the store. Feel free to navigate to https://www.chromiecraft.com/how-to-contribute/ and get to know, how to be a part of the server and get rewarded for it.

**Reply by BeefSteaker** [reply]
Ok but maybe more people would donate if it were more worth it.
I know I definitely wouldn't have bought the mount I did if I had know it wasn't account wide, and wouldn't buy any of the others currently (even though there are some I'd really like to)

**Reply by Amrx2** [reply]
You do already have a few mounts which are account wide available on the shop. As Honey said we need to make a balance between making Chromiepoints valuable to those who want to earn them via contributions, bug reports, events etc and those who just pay.

Of course as a non-profit organisation every penny earnt is put into the server to pay Devs for fixes so want to say thank you as well for your kind donation and hope you enjoy playing here on Chromiecraft!

**Reply by Aethereus-ManaOnion** [reply]
But the point is, that increasing the value of the product would drive sales and net more profit in the end. As @Amrx2 you said, every penny counts, but you seem hesitant to implement solutions to improve sales. Normally improving the value of a product requires investment, but as you are selling digital goods, it costs you virtually nothing to increase their worth because you dont have to produce them, you already have an unlimited supply of those mounts because they are just pixels. 

Our playerbase seems to be shrinking, and with it our customer base of people who would potentially be willing to but something from the shop, and the best thing about it is, that we can expand the offer to be more tempting to the existing players, effectively expanding the customer base FOR FREE. I myself play free to play games and regularily donate to them in exchange for cosmetics (Path of Exile). BUT, the major difference between their and our policy is that all the things you buy are permament. You buy a "transmog" for your armor? You can apply it on each and every one of your existing and new characters as long as the game is still standing. Hell, even if another version of the game will come out (POE2) we are told all our purchases will carry over. See, we don't buy stuff for a character, but for ourselves, which makes a colossal difference in the grand scheme of things...

Please tell me what you think.

**Reply by Amrx2** [reply]
Lets clear up some facts here:

> . As @Amrx2 you said, every penny counts

No I didn't say this at all. I said the following: _as a non-profit organisation every penny earnt is put into the server_. 

> Our playerbase seems to be shrinking

Incorrect, our overall numbers are up quite a bit over the last month or two and has continued to grow. I covered this on our discord channel a few days ago, our weekdays are busier than weekends were during the last bracket and week on week our average playercount continues to grow by double digit percentiles. 

>  increasing the value of the product would drive sales and net more profit in the end

Since when have we ever been about making profit or driving sales? We are a registered non-profit organisation (https://www.chromiecraft.com/en/privacy-policy-2/), if we were only driven by cash then we would ditch giving our hundreds of thousands of free chromiepoints each month, sell boosts, gear etc.  

> CC sounds absolutely demented by thinking that people will buy more mounts this way. 

Insults like these will get you absolutely nowhere. Again the primary point of the shop is to get people to contribute to the server, not to drive sales. 

We have answered this already and our stance on this hasn't changed from last, thanks.

**Reply by Aethereus-ManaOnion** [reply]
@Amrx2 I'm sorry for having sounded combative. The part that felt like an insult was an unfortunate and inappropriate figure of speech on my part. I apologize for making it sound like a personal insult. 

On the topic of the topic. Your argumentation sounds as if it's not in the best interest of Chromiecraft to increase sales. I always thought that more money would be better, even though you don't make profit personally yourselves, I thought that the point of having the shop was having more money to invest in the server and development. I didn't mean to imply that the staff is driven by profit, especially personal gain, as we all know that CC is non-profit.

> Again the primary point of the shop is to get people to contribute to the server, not to drive sales.

But the sales are part of how people contribute right? Making the offer more appealing would convince more people to donate and support the project financially. It's exactly the same reason why you offer services and products in return for donations instead of just ourtight asking people to support the server financially without anything but gratitude in return.

Please bear in mind that in a long-time, vocal, avid supporter and my goal is to help, provide feedback, perspective and insight into some issues. I never meant to just bitch and moan, or provide baseless, pointless criticism.

Again, I'm sorry for insulting you. That wasn't my goal when I was writing that post. I won't delete or edit it to seem like I never said it. I hope owning up to my words and apologizing will be enough to make you believe that my intentions are clear and I'm a friend not foe.

Thanks for the response and I'm looking forward to hearing more from you.

**Comment by BeefSteaker** [comment]
I understand why yall don't make them account wide, but personally it makes them not worth it for me. 

There's a bunch I'd have got to just use on random mount, but im not rich enough to buy them 9x, and I doubt you're throwing chromiepoints out at a rate that makes it a viable alternative

**Comment by Vyperhand** [comment]
I have to agree with this old suggestion.  There are SEVERAL items I would pay for if they were account wide that I will most likely NEVER pay for single-character at the rates being charged.  I mean, the vanity pets and mounts I'd be dropping cash on twice - once for my account, once for my wife's.  I honestly think you'd be looking at a net gain in donations if you were to make this change.  So, modifying the suggestion - why not try it for a month or two?  Look at the actual numbers it generates, and take action long-term based on how it works?

**Reply by Aethereus-ManaOnion** [reply]
You are 100% right on this, and CC sounds absolutely demented by thinking that people will buy more mounts this way. Why not make them 30$ more expensive? Because it would negatively impact sales.

**Reply by andovcr** [reply]
disagreeing with staff decisions is totally fair (I agree with you on wanting Bind-on-Account options for store pets/mounts)

using inflammatory and insulting language "sounds absolutely demented" is not going to persuade anyone of your point though
